app/src/pages/inside/logsPage/historyLine/historyLineItem/historyLineItemTooltip/historyLineItemTooltip.jsx

```diff
@@ -62,6 +66,12 @@ export const HistoryLineItemTooltip = injectIntl(
           {statusLocalization[status] ? formatMessage(statusLocalization[status]) : status}
         </span>
       </span>
+      <div className={cx('launch-name-block')}>
```

Comment:
This content in the tooltip should only appear when `includeAllLaunches` is `true`, otherwise all elements will have the same parent launch and don't need to be highlighted.
Can be done after [this](https://github.com/reportportal/service-ui/pull/2597#pullrequestreview-601540918)